### ISSUE TITLE
fix: loss in generalization after fine-tuning

### ISSUE THREAD

**Issue by florian-hoenicke** [jina-ai/now#76]
Finetuning on the test datasets with the current approach let's the model forget general knowledge.
If training data would perfectly represent the distribution of search queries in production, this would not be an issue. But this is not the case for our datasets (example: geolocation datset has country labels, but the user might insert something which is not a country).
Please fix this by forcing the linear head to keep the generalization ability.

Ideas (you can also try other things):
- use clip adapters to keep the zero shoot capability
https://arxiv.org/pdf/2110.04544.pdf

**Comment by florian-hoenicke** [comment]
we wait for finetuner coming up with something better.
